src/model/evaluate.py

```diff
@@ -21,57 +21,37 @@ def eval_model(model, loader: torch.utils.data.DataLoader, state: str, log: logg
     """
     model.eval()
     with torch.no_grad():
-        num_piece_classes = len(REVERSED_PIECE_TYPE)
-        num_piece_colors = len(REVERSED_PIECE_COLOR)
+        num_classes = len(LABELS)
 
-        type_correct_hits = torch.zeros(num_piece_classes)
-        type_counts = torch.zeros(num_piece_classes)
-        color_correct_hits = torch.zeros(num_piece_colors)
-        color_counts = torch.zeros(num_piece_colors)
+        class_correct_hits = torch.zeros(num_classes)
+        class_counts = torch.zeros(num_classes)
 
-        for i, (image, type_label, color_label, is_piece) in enumerate(loader):
-            type_pred_probs, color_pred_probs = model(image)
-            type_pred = torch.argmax(type_pred_probs, axis=1)
-            color_pred = torch.argmax(color_pred_probs, axis=1)
+        for i, (image, label) in enumerate(loader):
+            class_probs = model(image.to(device))
```

Comment:
What's the point of the exp here? This doesn't make it a probability. Softmax maybe may make more sense in this case, at least for clarity (although maybe it does the same)